client/helpers_test.go

```diff
@@ -968,3 +970,45 @@ func TestChangeTargetMetaFailsIfPrefixError(t *testing.T) {
 	require.Empty(t, repo.Targets[data.CanonicalTargetsRole].Signed.Targets)
 	require.Empty(t, repo.Targets["targets/level1"].Signed.Targets)
 }
+
+func TestAllNearExpiry(t *testing.T) {
+	repo, _, err := testutils.EmptyRepo("docker.com/notary")
+	require.NoError(t, err)
+	nearexpdate := time.Now().AddDate(0, 1, 0)
+	repo.Root.Signed.SignedCommon.Expires = nearexpdate
+	repo.Snapshot.Signed.SignedCommon.Expires = nearexpdate
+	repo.Targets["targets"].Signed.Expires = nearexpdate
+	_, err1 := repo.InitTargets("targets/exp")
+	require.NoError(t, err1)
+	repo.Targets["targets/exp"].Signed.Expires = nearexpdate
+	//Reset levels to display warnings through logrus
+	log.SetLevel(log.WarnLevel)
```

Comment:
nit: it'd be nice if this test (and the next one) reverted the log level upon finishing.  Can we do something like:

```
originalLogLevel := log.GetLevel()
// change to log warnings through logrus for testing purposes
log.SetLevel(log.WarnLevel)
// clean up log level upon test completion
defer log.SetLevel(originalLogLevel)
```